Source/santabs/SNTBundleService.m

```diff
@@ -25,6 +25,22 @@
 #import "SNTXPCConnection.h"
 #import "SNTXPCNotifierInterface.h"
 
+// For the SNTBundleService class, make SNTStoredEvent objects equal if the SHA256 matches.
+@implementation SNTStoredEvent (BundleEquatable)
```

Comment:
I'm not sure about this. Quoth the [Apple documentation](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/CustomizingExistingClasses/CustomizingExistingClasses.html):

> If the name of a method declared in a category is the same as a method in the original class, or a method in another category on the same class (or even a superclass), the behavior is undefined as to which method implementation is used at runtime.

The documentation used to be even stronger:

> Although the Objective-C language currently allows you to use a category to override methods the class inherits, or even methods declared in the class interface, you are strongly discouraged from doing so. A category is not a substitute for a subclass.

However, I'm not _exactly_ sure what a better solution looks like. 

- A subclass should work but then you're returning an array of the subclass rather than the stored event; this is fine unless the code using that array compares the objects in some way. 

- Another option would be to change relatedEvents on line 142 to a dictionary keyed by SHA-256 and only add to the dictionary if the key doesn't exist and then just return dict.allValues. You could avoid making the SNTStoredEvent objects at all this way instead of de-duping later.

Thoughts?